src/resolvers-cassandra/Settings/mutations.js

```diff
@@ -468,6 +518,7 @@ module.exports = {
   createSite: trackEvent(createSite, 'createSite'),
   removeSite: trackEvent(removeSite, 'removeSite'),
   createStream: trackEvent(createStream, 'createStream'),
+  editSite: trackEvent(withRunTime(editSite), 'editSite'),
```

Comment:
Is it worth adding `withRunTime` to track an operation to a single site or stream? If so, I'll need to add that to `createSite`, `removeSite`, `createStream`, `removeStream`